projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts

```diff
@@ -84,8 +84,8 @@ export function checkNotificationBannerOnOP(
   element,
   numberOfIssues?: number
 ): void {
-  element
-    .get('.cx-error-msg')
+  //element //Why use element instead of cy?
```

Comment:
Could you revert your changes and try to understand what the method does? You cannot change the API. This method is used in another tests that will fail.
In your tests you could use `verifyNotificationBannerOnOP` method that verifies whether the issues banner is displayed and the number of issues are accurate